python/rmm/tests/test_rmm.py

```diff
@@ -444,3 +444,34 @@ def test_mr_upstream_lifetime():
     # Delete cuda_mr first. Should be kept alive by pool_mr
     del cuda_mr
     del pool_mr
+
+
+@pytest.mark.skipif(
+    rmm._cuda.gpu.runtimeGetVersion() < 11020,
```

Comment:
I think technically we need to check both the runtime and driver version here. Someone could use a newer runtime with an older driver for example. Where the call would exist but would error at runtime.

Reply:
I'm a bit confused but happy to make the change.

Reply:
cudaMallocAsync depends on having both libcudart >= 11.2 and libcuda >= 11.2. If say you have libcudart == 11.2 and libcuda == 11.0, then https://github.com/rapidsai/rmm/blob/branch-0.19/include/rmm/mr/device/cuda_async_memory_resource.hpp#L49 would error at runtime that there isn't a new enough driver for the feature. If you had libcudart == 11.0 and libcuda == 11.2, then https://github.com/rapidsai/rmm/blob/branch-0.19/include/rmm/mr/device/cuda_async_memory_resource.hpp#L49 would error with an invalid DeviceAttribute since it doesn't exist in libcudart 11.0.